gap/semigroups/semitrans.gi

```diff
@@ -741,6 +741,82 @@ function(S)
   return t;
 end);
 
+InstallMethod(WreathProduct,
+"for a transformation monoid and a permutation group",
+[IsTransformationMonoid, IsPermGroup],
+function(M, G)
+  local S;
+  S := AsMonoid(IsTransformationMonoid, G);
```

Comment:
This could just be `return WreathProduct(M, AsMonoid(IsTransformationMonoid, G))`